feat(Forms): add `requireCommit` property to Iterate.PushContainer to show error during submit or next step after user input (#5005)
dnbexperience/eufemia: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/Iterate/PushContainer/info.mdx

Good to know:

- Fields inside the container must have an `itemPath` defined, instead of a `path`.

> - As soon as the user has entered data, the PushContainer will show an error message if they try to submit the form or navigating to the next step in a Wizard.

Reviewer: ```suggestion
- As soon as the user has entered data, the PushContainer will show an error message if user try to submit the form or navigate to the next step in a Wizard.
```

> - As soon as the user has entered data, the PushContainer will show an error message if they try to submit the form or navigating to the next step in a Wizard.

Reviewer: ```suggestion
- When user has entered data without committing it in a PushContainer, and tries to submit the form or navigating to the next step in a Wizard, the PushContainer will show an error message.
```

> - When the user has entered data without committing it (to the outer context), and tries to submit the form or navigate to the next step in a wizard, the PushContainer will display an error message.”

Reviewer: ```suggestion
- When the user has entered data without committing it (to the outer context), and tries to submit the form or navigate to the next step in a wizard, the PushContainer will display an error message.
```

Looking great now 👏
